kubernetes/bin/deploy-development.sh

```diff
@@ -1,27 +1,51 @@
 #!/bin/bash
 export DIR=kubernetes
 
-USAGE="USAGE: ./kubernetes/bin/deploy-development.sh create|delete"
+USAGE="USAGE: ./kubernetes/bin/deploy-development.sh create|create all|delete|delete all"
 
-if [ "$1" = "create" ]; then
+if [[ "$1" = "create" ]];
+then
     echo "Creating objects from configurations."
     echo "Make sure that persistent disk is already created."
     echo ""
     echo "Creating Elasticsearch Replication Controller"
-    kubectl create -R -f ${DIR}/yamls/development/elasticsearch/
-    echo "Waiting for server to start up. ~20s."
-    sleep 20
+        kubectl create -R -f ${DIR}/yamls/staging/elasticsearch/
+    # For mqtt
+        kubectl create -R -f ${DIR}/yamls/mosquitto/
+    if [[ "$2" = "all" ]];
+    then
+        # Start KubeLego deployment for TLS certificates
+        kubectl create -R -f ${DIR}/yamls/lego/
+        echo "Start nginx deployment, ingress & service"
+        kubectl create -R -f ${DIR}/yamls/nginx/
+    fi
+    # Create web namespace
+    kubectl create -R -f ${DIR}/yamls/staging/web/
     echo "Creating loklak deployment"
-    kubectl create -R -f ${DIR}/yamls/development/api-server/
-    echo "Trying to fetch public IP. ~40s."
-    sleep 40
-    kubectl get services --namespace=web
+    # Create API server deployment and service for development branch
+    kubectl create -R -f ${DIR}/yamls/staging/api-server/
```

Comment:
Yes, it would work most of the times. But due to some reasons, if the api-server is created before elasticsearch, it would throw error as `No nodes matching...` and even fail to create indices if it is a fresh deployment.